### ISSUE TITLE
buffer support

### ISSUE THREAD

**Issue by balupton** [bevry/make-deno-edition#9]
perhaps https://www.npmjs.com/package/buffer can be imported, although it is not typescript and has two deps, so prob won't work with deno

deno are working on a compat layer https://deno.land/std/node at https://deno.land/std/node/buffer.ts

**Comment by balupton** [comment]
buffer imports landed in v0.5.0

buffer globals under consideration

**Comment by balupton** [comment]
going to recommend people do:

``` typescript
import type Buffer from 'buffer'
```

or

```
import Buffer from 'buffer'
```

as that is way easier for me
